picasso/src/main/java/com/squareup/picasso3/Picasso.java

```diff
@@ -397,7 +402,7 @@ public void invalidate(@NonNull File file) {
   }
 
   /** {@code true} if debug indicators should are displayed on images. */
-  @SuppressWarnings("UnusedDeclaration") public boolean areIndicatorsEnabled() {
+  @SuppressWarnings("UnusedDeclaration") public boolean getIndicatorsEnabled() {
```

Comment:
revert?  boolean getter?

Reply:
This is a failure.

https://android.github.io/kotlin-guides/interop.html#property-prefixes

Reply:
and `isIndicatorsEnabled` is weird and broken english